terraform/environments/mlra/modules/ALB/variables.tf

```diff
@@ -0,0 +1,144 @@
+variable "account_number" {
+  type        = string
+  description = "Account number of current environment"
+}
+variable "tags" {
+  type        = map(string)
+  description = "Common tags to be used by all resources"
+}
+variable "application_name" {
+  type        = string
+  description = "Name of application"
+}
+variable "public_subnets" {
+  type        = list(string)
+  description = "Public subnets"
+}
+# variable "loadbalancer_ingress_rules" {
+#   description = "Security group ingress rules for the loadbalancer"
+#   type = map(object({
+#     description     = string
+#     from_port       = number
+#     to_port         = number
+#     protocol        = string
+#     security_groups = list(string)
+#     cidr_blocks     = list(string)
+#   }))
+# }
+
+# variable "loadbalancer_egress_rules" {
+#   description = "Security group egress rules for the loadbalancer"
+#   type = map(object({
+#     description     = string
+#     from_port       = number
+#     to_port         = number
+#     protocol        = string
+#     security_groups = list(string)
+#     cidr_blocks     = list(string)
+#   }))
+# }
```

Comment:
Is any of this commented block required?

Reply:
This has been deleted